pkg/unstruct/v1alpha2/unstruct.go

```diff
@@ -78,3 +80,57 @@ func (b *builder) Build() (*unstruct, error) {
 	}
 	return b.unstruct, nil
 }
+
+// unstructList contains a list of unstructured
+// items
+type unstructList struct {
+	items []*unstruct
+}
+
+// listBuilder enables building a list
+// of an unstruct instance
+type listBuilder struct {
+	list unstructList
+	errs []error
+}
+
+// ListUnstructBuilderForYamls returns a mew instance of
+// list unstruct builder by making use of the provided YAMLs
+func ListUnstructBuilderForYamls(docs string) *listBuilder {
+	lb := &listBuilder{}
```

Comment:
IMO `ListBuilderForYamls` will be better name as package name is `unstruct` itself.. make the linter happy.

Reply:
correct @prateekpandey14